features/steps/toolpanel.js

```diff
@@ -0,0 +1,31 @@
+/* This file is part of Ezra Bible App.
```

Comment:
To be consistent I would name the file `tool_panel.js` instead of `toolpanel.js`.